_tutorials/npm.md

```diff
@@ -26,6 +26,8 @@ If migrating from Bower to npm, [please refer to our migration guide](/docs/tuto
 
 ## How to avoid multiple versions of Origami in the dependency tree
 
+Use npm v7 and Origami specification version 2 components.
```

Comment:
I think this tutorial, and the related npm tutorial, should be updated more dramatically. They should probably be merged into one. With the main recommendation being to upgrade components, with a link to the spec v2 migration guide. 

And a trimmed down section with current content in case anyone is curious about them. Explaining that some v1 components are on npm, we don't recommend them since they don't maintain a flat dependency tree, but there is `is-origami-flat` if you want

Thoughts?

Reply:
>with a link to the spec v2 migration guide

that would be another great addition to this PR or a follow up :)

Reply:
> > with a link to the spec v2 migration guide
> 
> that would be another great addition to this PR or a follow up :)

Just to confirm I understand, you are suggesting we write a generic migration guide for Origami Component Authors/Maintainers on how to migrate from spec v1 to spec v2?

Reply:
Yeah that was my thinking

Reply:
Good idea :-)